packages/docs/.vuepress/public/images/anatomy-modal.svg

```diff
@@ -1,5 +1,4 @@
 <svg width="958" height="455" viewBox="0 0 958 455" fill="none" xmlns="http://www.w3.org/2000/svg">
-<rect width="958" height="455" fill="#EBEBED"/>
```

Comment:
Did this get removed by running SVGO? Curious why this is only now a problem.

Reply:
Joe had baked in a background to the image, so I removed it. By inlining the SVG in a div, it created gaps in the Visual component. I added a fill prop to that component which provides the background color for us.